config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
+
+HypervisorPath is an optional absolute path to the hypervisor binary used by VM-based runtimes.
+
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
```

Comment:
On Wed, Apr 27, 2016 at 07:46:13AM -0700, Lai Jiangshan wrote:
> > +## ImagePath
> > +
> > +ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
> 
> When `ImagePath` and `root.path` are both specific, the rootfs  will the the
> `filepath.Join(ImageMountPoint, root.path)`  (root.path can be absolute or not)
> the `ImageMountPoint` is mounted by the guest inside the VM. the `ImagePath` is the block device passed to VM by the hypervisor.

I think you mean ‘ImagePath’ where you have ‘ImageMountPoint’ in your
comment.

If the runtime needs to pass a block device to the VM, it seems like
it could expand root.path into an absolute path, and then keep running
dirname or similar until it found a block-device mount-point.  Is
there really a need to split it into two fields?  See also #133, which
sounds similar.